en_US/advanced/cluster.md

```diff
@@ -309,12 +309,35 @@ cluster.autoclean = 5m
 ```
 
 ## Firewall settings
-If there is a firewall between the cluster nodes, the firewall needs to open port 4369 and a TCP port segment. 4369 is used by the epmd port mapping service. The TCP port segment is used to establish connections and communications between nodes.
+If the environment variable WITH_EPMD=1 is set in advance, the epmd (listening port 4369) will be enabled for node discovery when emqx is started, which is called `epmd mode`.
+If the environment variable WITH_EPMD is not set, epmd is not enabled when emqx is started, and emqx ekka is used for node discovery, which is also the default method of node discovery  after version 4.0. This is called `ekka mode`.
 
-After the firewall is set, you need to configure the same port segment in  `emqx/etc/emqx.conf`:
+**epmd mode：**
+
+If there is a firewall between cluster nodes, the firewall needs to open TCP port of 4369 and a TCP port segment. 4369 is used by the epmd port for mapping service. The TCP port segment is used to establish connection and communication between nodes.
```

Comment:
I'd recommend to rephrase like this
```suggestion
If there is a firewall between cluster nodes, the firewall needs to open TCP port 4369 for each node, to allow peers query each other's listening port. The firewall should also allow nodes connecting to port in configurable range from `node.dist_listen_min` to `node.dist_listen_max` (inclusive, default is `6369` for both) 
```

then there is no need for the lines from 319 to 325